csp-mixin/panels.libsonnet

```diff
@@ -899,5 +899,636 @@ local commonlib = import 'common-lib/common/main.libsonnet';
       + g.panel.timeSeries.fieldConfig.defaults.custom.stacking.withMode('normal')
       + this.signals.azurevirtualnetwork.tcpPacketsForwarded.asPanelMixin()
       + this.signals.azurevirtualnetwork.udpPacketsForwarded.asPanelMixin(),
+
+    // Gcp Compute Engine
+
+    gce_instance_count:
+      this.signals.gcpceOverview.instanceCount.asStat()
+      + g.panel.table.standardOptions.withOverrides([
+        {
+          matcher: {
+            id: 'byFrameRefID',
+            options: 'Instance count',
+          },
+          properties: [
+            {
+              id: 'mappings',
+              value: [],
+            },
+            {
+              id: 'unit',
+              value: 'short',
+            },
+            {
+              id: 'thresholds',
+              value: {
+                mode: 'absolute',
+                steps: [
+                  {
+                    color: 'text',
+                    value: null,
+                  },
+                ],
+              },
```

Comment:
I think you may be doing this in order to change the stats text color to "gray", but it doesn't appear to be taking effect.

I achieved the same thing in the queue storage dashboard using [this solution](https://github.com/grafana/jsonnet-libs/blob/547934640d91d6ed08e69229c5f6a4ec6e3ed250/csp-mixin/panels/azurequeuestore.libsonnet#L66-L69).

Reply:
Thank you, I like better your solution! https://github.com/grafana/jsonnet-libs/pull/1336/commits/94adb658561f72444ac7851c70e070dda64d55cd

Reply:
JFYI, you could also add default commonlib style to stat panels, to avoid seeing everything in green(<80) and red(=>80):
like so:
```
      this.signals.gcpceOverview.instanceCount.asStat()
      + commonlib.panels.generic.stat.base.stylize()
```
It has the same default style of color text, not green/red thresholds.
https://github.com/grafana/jsonnet-libs/blob/master/common-lib/common/panels/generic/stat/base.libsonnet